docs/nodes/blockspace-race.mdx

```diff
@@ -201,6 +202,7 @@ broadcast transactions.
 * [https://grpc.blockspace-race.celestia.counterpoint.software/](https://grpc.blockspace-race.celestia.counterpoint.software/)
 * [https://grpc.celestia.testnet.injective.network/](https://grpc.celestia.testnet.injective.network/)
 * [https://cumulo.org.es/celestia.grpc/](https://cumulo.org.es/celestia.grpc/)
+* [celestia-grpc.stakin-nodes.com:443](celestia-grpc.stakin-nodes.com)
```

Comment:
is this http or https? please add this and a trailing slash, thanks!

Reply:
checking in here @gle1pn1r

Reply:
Hey, just to be sure these are supposed to be regular gRPC endpoints, not gRPC-web correct? Usually these are targeted as hostname:port.

Reply:
Merge conflicts resolved and endpoints changed to https://